src/utils/amplitude.ts

```diff
@@ -19,6 +20,19 @@ export const logPageVisit = (url: string, sideTittel: string) => {
   client.logEvent(EventType.PageView, { url: url, sidetittel: sideTittel });
 };
 
+export const logEvent = (eventType: EventType, data: any) => {
```

Comment:
Satt og tenkte på om det var noen måte å unngå `any` her, men det er kanskje ikke så lett 🤔 Evt en eller annen union-type som er knyttet til `EventType`?
At hvis det er av typen `EventType.ButtonClick` så kommer `data` til å ha propertien `tekst`.

Reply:
https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions
Noe sånt kanskje:
```
type EventPageView = {
  type: EventType.PageView;
  data: {
    url: string;
    sideTittel: string;
  };
};

type EventButtonClick = {
  type: EventType.ButtonClick;
  data: {
    tekst: string;
  };
};

type Event = EventPageView | EventButtonClick;
export const logEvent = (event: Event) => {...}
```